### ISSUE TITLE
Compilation fails on OSX 

### ISSUE THREAD

**Issue by habemus-papadum** [libfuse/python-fuse#14]
Installing via `pip install fuse-python`
I receive the following compilation error:

```
fuseparts/_fusemodule.c:951:23: error: use of undeclared identifier '_IOC_WRITE'; did you mean 'UIO_WRITE'?
            if(!(_IOC_DIR(cmd) & _IOC_WRITE)) {
                                 ^~~~~~~~~~
                                 UIO_WRITE
    /usr/include/sys/uio.h:90:25: note: 'UIO_WRITE' declared here
    enum uio_rw { UIO_READ, UIO_WRITE };
                            ^
    fuseparts/_fusemodule.c:963:21: error: use of undeclared identifier '_IOC_READ'; did you mean 'UIO_READ'?
            if(_IOC_DIR(cmd) & _IOC_READ) {
                               ^~~~~~~~~
                               UIO_READ
    /usr/include/sys/uio.h:90:15: note: 'UIO_READ' declared here
    enum uio_rw { UIO_READ, UIO_WRITE };
```

**Comment by xdgc** [comment]
This seems to have arrived with 26655db1, which implements ioctl in a decidedly non-portable fashion. I'm able to build if I "update" to 7e29c2ae first, and it works on this Mac with the "hello.py" filesystem.

**Comment by mezantrop** [comment]
I have created a [pull-reques](https://github.com/libfuse/python-fuse/pull/15) to address the issue with the same symptoms on FreeBSD.

**Comment by henryborchers** [comment]
Same problem on my Mac. Has anybody reviewed the pull request?

**Comment by Rajczyk** [comment]
Same problem here too

**Comment by sdelafond** [comment]
This is now fixed in master by 1d80a577dc3d38baad68a35c7d94e7d1ac1a101d, thanks to @mezantrop.
